Dockerfile

```diff
@@ -43,4 +43,13 @@ RUN
 # Append to /etc/profile for login shells
 RUN echo 'export PATH="/usr/local/go/bin:$PATH"' >> /etc/profile
 
+# Create .cargo directory
+RUN mkdir -p /root/.cargo
+
+# Install rustup and cargo
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
```suggestion
RUN wget -nv -O- https://sh.rustup.rs | sh -s -- -y
```

Reply:
In addition can we also specify exact toolchain version? For more reproducible builds